### PR TITLE
Portfolio menu

### PR DESCRIPTION
**Pull request by hiwina**
Some of the things done in this branch are

- [ ] added the mobile menu in the index.html

- [ ] added styles for the menu

- [ ] added javaScript file for listening events

### CODE REVIEW

**Review by ElsonOtake — COMMENTED** [review]
Hi @hiwina, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid code review request

## Invalid Code Review Request

You have submitted a project, that does not have commits from the coding partner. It's a pair programming activity. There must be a change record of all involved in the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @hiwina,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

### Invalid Code Review Request

You have submitted a project, that for Portfolio: details popup window activity on dashboard but you implemented the project for Portfolio:Mobile Menu

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request does not count into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -9,15 +9,30 @@
 </head>
```
Comment (omarsalem7) [inline index.html:10]: - [ ] Kindly add the following meta tag `<meta name="viewport" content="width=device-width, initial-scale=1" />` inside of `<head>`, so when I click on the mobile toggle icon, I can see the mobile version, not the desktop also as shown in the following image.
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @hiwina, @edenlisk 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @hiwina      :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**